Konvoy release docs (#3521)
mesosphere/dcos-docs-site: pages/dkp/konvoy/1.8/install/http-proxy/index.md

      noProxy: [ "172.0.0.0/16", ".intra.net" ]
```

> In a cluster with the default configuration, the `kommander` and `gatekeeper` addons are installed. Gatekeeper will automatically configure the right proxy settings to `kommander`. If on the other hand, `gatekeeper` is not enabled then you will have to manually configure `kommander`.

Reviewer: ```suggestion
In a cluster with the default configuration, the `kommander` and `gatekeeper` addons are installed. Gatekeeper will automatically configure the right proxy settings for `kommander`. If `gatekeeper` is not enabled then you will have to manually configure the proxy settings for `kommander`.
```
